### ISSUE TITLE
Publish http4s scalafix rules

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#6239]
We're losing the scalafix rules as we split the repo.  We need to figure out how to refer to them here and get them into module.g8, or publish them as a jar.

**Comment by armanbilge** [comment]
Right. Then ideally we get scalafix integrated into sbt-typelevel.
- https://github.com/typelevel/sbt-typelevel/issues/236

**Comment by armanbilge** [comment]
I can work on this one.
